packages/Form/summary/src/Summary.tsx

```diff
@@ -30,9 +30,9 @@ const Summary = ({
       icon="warning-sign"
       title={title}>
       <ul className="af-summary__message-list">
-        {messages.map((message) => (
-          <li className="af-summary__message-item" key={message}>
-            <span>{message}</span>
+        {messages.map((message, index) => (
+          <li className="af-summary__message-item" key={index}>
```

Comment:
c'est possible d'utiliser autre chose que l'index pour les keys ? D'après la doc, ce n'est pas préconisé : https://fr.reactjs.org/docs/lists-and-keys.html#keys

Reply:
@samuel-gomez-axa oui pas de soucis, est-ce qu'une concaténation du type key={`message_${index}`} pourrait convenir à ton avis ?

Reply:
Utilisation du createId pour avoir un id unique.